src/rpm/builder.rs

```diff
@@ -707,6 +707,20 @@ impl RPMBuilder {
         }
 
         let offset = 0;
+        let (small_package, combined_file_sizes_entry) = match combined_file_sizes.try_into() {
```

Comment:
I would still rather the small package check be separate personally.

Reply:
Then we would have `combined_file_sizes.try_into().unwrap()` later in code, which is implicitly connected to the `small_package` calculation. In this code the connection is explicit and robust, for example, if hypothetically you will switch from `u32` to `i32` in future, it will continue to work correctly, while with the separate calculation you would have to remember to update it.

Either way, I don't care about it too much, so I can change it if you like it more.